src/homework4/gaze_to_the_past.py

```diff
@@ -0,0 +1,19 @@
+"""
+Даны два натуральных числа.
+Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+a = 1071
+b = 462
+
+while a and b:
+    if a > b:
+        a = a % b
+    else:
+        b = b % a
+# if module not a zero, it is common delimiter
+if a > b:
```

Comment:
it's a trick cause you don't know where is the bigger one. it can be made in other way without these hacks